src/main/java/com/Elessar/app/server/LogOnHandler.java

```diff
@@ -42,7 +45,9 @@ public void handle(HttpExchange he) throws IOException {
             final String userName = logonRequest.getName();
             final String password = logonRequest.getPassword();
 
+            monitor.timerStart(MyDatabase.UPDATE);
```

Comment:
It's better to use something like "db.user.update" instead of "update" as the operation. Otherwise how do we distinguish different database or different collection?